spec/features/orders_form_spec.rb

```diff
@@ -19,6 +19,16 @@
     )
   end
 
+  describe "belongs_to relationships" do 
+    it 'uses the stored value as the selected value' do 
+      order = create(:order)
+      customer = order.customer
+
+      visit edit_admin_order_path(order)
+      expect(page).to have_select('Customer', selected: customer.name)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.